index.js

```diff
@@ -4,12 +4,29 @@
 module.exports = {
     name: 'ember-cli-hot-loader',
     serverMiddleware: function (config){
-        var lsReloader = require('./lib/hot-reloader')(config.options);
-        lsReloader.run();
+        if (config.options.environment === 'development') {
```

Comment:
Looks like at the very least we need this for tests as well before merging this, see https://travis-ci.org/toranb/ember-cli-hot-loader/jobs/205835258